src/test/java/edu/ucla/library/iiif/fester/fit/GetStatusFT.java

```diff
@@ -47,4 +51,9 @@ public final void testStatus(final TestContext aContext) {
         });
     }
 
+    @FunctionalInterface
```

Comment:
What's the reason for using an interface, instead of just doing a direct `String.split()`?

Reply:
I think it's because the code checks would fail whenever I did String.spilt(" ") because " " was repeated three times. I could have just put " " in a variable instead, but I had never written an interface so I wanted to try that. If you think it's better though I can change it to use String.split()

Reply:
FWIW, I think the constant and `String.split()` would be simpler to maintain... but perhaps Mark or Kevin will convince me otherwise.

Reply:
The checkstyle rule complains about the multiple " "s because they are allocating multiple strings for the same value. We have a `Constants.EMPTY` variable defined for " " so you can just use that. You can also use a static import to get that down to just `EMPTY` in the code.

Reply:
On reread; two points: 1) since you're using MB_STR in both handler and test, should probably be added to Constants.java, for consistency/simplciity; 2) you probably don't need the split code--just do a `String.contains()` for the constant.

Reply:
Sounds good. Is there somewhere specific I should add it in the file or do I just add it to the end? I'll change it to String.contains as well. That makes a lot more sense.

Reply:
Oops, my mistake... that should have been Constants.SPACE, but that's only defined in the newer version of freelib-utils. For now, it's probably easier to just put a new SPACE variable (" ") in Fester's Constants class.

Reply:
I think I finished fixing everything so that it follows all of your suggestions.